component/file_cache/lru_policy.go

```diff
@@ -275,6 +283,7 @@ func (p *lruPolicy) cacheValidate(name string) {
 // For all other timer based activities we check the stuff here
 func (p *lruPolicy) clearCache() {
 	log.Trace("lruPolicy::ClearCache")
+	defer p.wg.Done()
```

Comment:
In reason to wait for this thread is to clear the cache before the unmount goes out ?